### PR TITLE
Add live tests

### PR DESCRIPTION
**Pull request by PrasRsRos**
Add template for live test using D455 camera that supports changing of parameters dynamically

### CODE REVIEW

File: realsense2_camera/test/live_camera/test_camera_all_profile_tests.py [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+            cap = pytest_live_camera_utils.get_camera_capabilities(params['device_type'], serial_no)
+            if cap == None:
+                print("Device not found? : " + params['device_type'])
+                return
+            self.create_param_ifs(params['camera_name'] + '/' + params['camera_name'])
+            for profile in cap["color_profile"]:
+                if profile[0] == 'Color':
+                    print("Testing :"+ profile[0] + " " + profile[1] + " " + profile[2])
+                    if "BGRA8" == profile[2]:
+                        print("Skipping " +profile[2])
```
Comment (SamerKhshiboun) [inline realsense2_camera/test/live_camera/test_camera_all_profile_tests.py]: I added support for these type of formats in our node.
Still in review.. maybe when we merge my PR you will remove this "skipping"

**Comment by SamerKhshiboun** [comment]
Hi @PrasRsRos

I reviewed your PR, and it looks good.
Do you prefer to wait with this PR till we merge my PR (https://github.com/IntelRealSense/realsense-ros/pull/2840) ?
After merging my PR, most of skipping tests should work in your side, and I think you should unskip these tests then.
If you want to handle these fixes of skips in another PR, let me know.
Meanwhile, I'm waiting for review on my PR.

File: realsense2_camera/test/live_camera/test_camera_all_profile_tests.py [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+    elif profile == 'Infrared':
+        if "Y8" == format:
+            return True
+        if "Y16" == format:
+            return True
+        if "BGRA8" == format:
+            return True
+        if "RGBA8" == format:
+            return True
+        if "Y10BPACK" == format:
```
Comment (SamerKhshiboun) [inline realsense2_camera/test/live_camera/test_camera_all_profile_tests.py:61]: on which camera model do you see this format ?

Comment (Arun-Prasad-V) [reply realsense2_camera/test/live_camera/test_camera_all_profile_tests.py:61]: in D415:     
- Infrared 1:	  1920x1080	@ 30Hz	   Y10BPACK

**Comment by PrasRsRos** [comment]
Imu test fails when trying to change enable_accel & enable_gyro dynamically.

**Comment by SamerKhshiboun** [comment]
> Imu test fails when trying to change enable_accel & enable_gyro dynamically.

on specific model ? or on all cameras ?

**Comment by PrasRsRos** [comment]
> > Imu test fails when trying to change enable_accel & enable_gyro dynamically.
> 
> on specific model ? or on all cameras ?

I could test only on D455. However, it could be an issue with my understanding also. So a review of the code helps before checking the code.

**Comment by PrasRsRos** [comment]
New pull request: https://github.com/IntelRealSense/realsense-ros/pull/2865

**Pull request closed without merging**